tools/clang/test/CodeGenSPIRV/spirv.legal.sbuffer.struct.vulkan1.2.hlsl

```diff
@@ -0,0 +1,33 @@
+// RUN: %dxc -T cs_6_3 -fspv-target-env=vulkan1.2 -E main
```

Comment:
Please rename this file as "tools/clang/test/CodeGenSPIRV/spirv.legal.sbuffer.struct.vulkan1p2.hlsl"

Reply:
Done